selfservice/flow/saml/handler.go

```diff
@@ -0,0 +1,336 @@
+package saml
```

Comment:
To follow the rest of the design pattern, this handler should move into the strategy. I don't think that it belongs into the flow as it doesn't have its own flow object nor any other properties of a flow (hooks, ui, ...) as far as I can tell.